src/_client.ts

```diff
@@ -76,6 +76,13 @@ export interface ChalkClientOpts {
    */
   fetchHeaders?: typeof Headers;
 
+  /**
+   * The format to use for date-type data.
+   *
+   * Defaults to "ISO_8601" (in UTC), also supports "EPOCH_MILLIS"
+   */
+  timestampFormat?: ChalkClientConfig["timestampFormat"];
```

Comment:
Referring to the `ChalkClientConfig`, which is really only intended for internal consumption with the client, seems kind of odd and roundabout to me. I think it would be simpler to just type this as 
```ts
{
  timestampFormat?: TimestampFormat;
}
```